doc/source/whatsnew/v0.24.2.rst

```diff
@@ -59,7 +59,7 @@ Bug Fixes
 
 - Bug in :meth:`~pandas.core.groupby.GroupBy.transform` where applying a function to a timezone aware column would return a timezone naive result (:issue:`24198`)
 - Bug in :func:`DataFrame.join` when joining on a timezone aware :class:`DatetimeIndex` (:issue:`23931`)
-- Bug in :func:`DataFrame.pivot_table` where use :class:`IntervalIndex` as pivot index would raise `TypeError` (:issue:`25814`)
+- Bug in :func:`DataFrame.pivot_table` where use :class:`IntervalIndex` as pivot index would raise ``TypeError`` (:issue:`25814`)
```

Comment:
Change "where use" -> "with a"

Reply:
Ok. I just realized this entry should be added to v0.25.0 whatsnew, right ?

Reply:
yes, this should be in the 0.25.0 whatsnew